ledger/participant-integration-api/src/main/scala/platform/apiserver/configuration/RateLimitingConfig.scala

```diff
@@ -6,11 +6,13 @@ package com.daml.platform.apiserver.configuration
 final case class RateLimitingConfig(
     maxApiServicesQueueSize: Int,
     maxApiServicesIndexDbQueueSize: Int,
+    maxStreams: Int,
```

Comment:
The second two have been renamed.

When it comes to the first:
Everything is 'Api' => maxActiveStreamingCalls
This will limit both active _and_ inactive calls => maxStreamingCalls
Everything is a call => maxStreams   ;-)